src/flamegpu/model/ModelData.cpp

```diff
@@ -172,3 +172,10 @@ bool ModelData::hasSubModelRecursive(const std::shared_ptr<const ModelData> &sub
     return false;
 }
 
+ModelData::size_type ModelData::getMaxLayerWidth() const {
+    unsigned int maxWidth = 0u;
+    for (auto &layer : layers) {
+        maxWidth = std::max(maxWidth, static_cast<ModelData::size_type>(layer->agent_functions.size()));
```

Comment:
Wrap `std::max` in brackets e.g. `(std::max)`.

VC++ doesn't like it because `windows.h` must be included somewhere, and this declared `min` and `max` as pre-processor macros.